tests/IBFE/explicit_ex5.cpp

```diff
@@ -732,7 +732,7 @@ postprocess_data(Pointer<Database> input_db,
         // This computed value is not stable between debug and optimized mode
         // (and also different versions of libMesh): make it slightly more
         // stable by increasing the order of magnitude
-        pout << loop_time << " " << 0.01 + -F_integral[1] / (0.5 * rho * U_max * U_max * D) << endl;
+        pout << loop_time << " " << F_integral[1] << endl;
```

Comment:
Is the comment still correct?

Reply:
Maybe also use the dimensional expression for `F_integral[0]` too?

Reply:
No (I will fix it) and good idea - I'll change that as well.

Reply:
The extra factors for the first term were equivalent to multiplying by 0.5, which in turn multiplied the error by 0.5 - as a result things now pass that once failed. I'm working on it.

Reply:
Fixed!